etl_project/assets/extract_news.py

```diff
@@ -174,3 +173,43 @@ def download_from_s3(
     return df
 
 
+
+def calculate_word_frequency(article_contents, word):
+    """Calculates number of times a word appears in the article"""
+    return article_contents.lower().count(str(word).lower())
+
+def process_articles(word_by_grade_level_data, newspaper_articles_data):
```

Comment:
in process_articles(), what am I supposed to input for the arguments? I've been trying to make different inputs for newspaper_article_data and am not sure what the input should be.